### ISSUE TITLE
declarationSubject - make this not a required field 

### ISSUE THREAD

**Issue by kathryn-ods** [openownership/data-standard#728]
I think declarationSubject might not make sense as a required field in all contexts. 

E.g. if you had a consolidated register where one consolidated person/company statement is linked to various declaring companies it might not be obvious what this field should be?

**Comment by kd-ods** [comment]
Yes - I was wondering whether making it required on relationship statements only would work.
